RecoParticleFlow/PFProducer/python/particleFlowSimParticle_cfi.py

```diff
@@ -8,16 +10,9 @@
     # replace ParticleFilter.pTMin = 0.5
     # flags 
     process_RecTracks = cms.untracked.bool(False),
-    ParticleFilter = cms.PSet(
-        EProton = cms.double(5000.0),
-        # Particles with |eta| > etaMax (momentum direction at primary vertex) 
-        # are not simulated 
-        etaMax = cms.double(5.0),
-        # Charged particles with pT < pTMin (GeV/c) are not simulated
-        pTMin = cms.double(0.0),
-        # Particles with energy smaller than EMin (GeV) are not simulated
-        EMin = cms.double(0.0)
-    ),
+    #
+    ParticleFilter = ParticleFilterBlock.ParticleFilter,
```

Comment:
I'm not sure that this leads to a full copy of FastSimulation.Event.ParticleFilter_cfi.ParticleFilterBlock 
Please use a copy or make sure that the edits applied below are not modifying the values at the source.

perhaps 
```python
ParticleFilter = ParticleFilterBlock.ParticleFilter.clone(chargedPtMin = 0, EMin = 0)
```
is a simpler alternative (I didn't check that this syntax works)